compendium/dnd5e.items.json

```diff
@@ -1,2407 +1,2408 @@
-            "name": ""
-        },
-        "Wand": {
-            "name": ""
-        },
-        "Wand of Binding": {
-            "name": ""
-        },
-        "Wand of Enemy Detection": {
-            "name": ""
-        },
-        "Wand of Fear": {
-            "name": ""
-        },
-        "Wand of Fireballs": {
-            "name": ""
-        },
-        "Wand of Lightning Bolts": {
-            "name": ""
-        },
-        "Wand of Magic Detection": {
-            "name": ""
-        },
-        "Wand of Magic Missiles": {
-            "name": ""
-        },
-        "Wand of Paralysis": {
-            "name": ""
-        },
-        "Wand of Polymorph": {
-            "name": ""
-        },
-        "Wand of Secrets": {
-            "name": ""
-        },
-        "Wand of Web": {
-            "name": ""
-        },
-        "Wand of Wonder": {
-            "name": ""
-        },
-        "Wand of the War Mage +1": {
-            "name": ""
-        },
-        "Wand of the War Mage +2": {
-            "name": ""
-        },
-        "Wand of the War Mage +3": {
-            "name": ""
-        },
-        "War Pick": {
-            "name": ""
-        },
-        "War Pick +1": {
-            "name": ""
-        },
-        "War Pick +2": {
-            "name": ""
-        },
-        "War Pick +3": {
-            "name": ""
-        },
-        "Warhammer": {
-            "name": ""
-        },
-        "Warhammer +1": {
-            "name": ""
-        },
-        "Warhammer +2": {
-            "name": ""
-        },
-        "Warhammer +3": {
-            "name": ""
-        },
-        "Waterskin": {
-            "name": ""
-        },
-        "Weaver's Tools": {
-            "name": ""
-        },
-        "Well of Many Worlds": {
-            "name": ""
-        },
-        "Whetstone": {
-            "name": ""
-        },
-        "Whip": {
-            "name": ""
-        },
-        "Whip +1": {
-            "name": ""
-        },
-        "Whip +2": {
-            "name": ""
-        },
-        "Whip +3": {
-            "name": ""
-        },
-        "Wind Fan": {
-            "name": ""
-        },
-        "Winged Boots": {
-            "name": ""
-        },
-        "Wings of Flying": {
-            "name": ""
-        },
-        "Woodcarver's Tools": {
-            "name": ""
-        },
-        "Wooden Staff": {
-            "name": ""
-        },
-        "Yew Wand": {
-            "name": ""
-        }
-    },
-    "label": "Gegenstände (SRD)",
-    "mapping": {
-        "name": {
-            "converter": "itemname",
-            "path": "name"
-        },
-        "rarity": {
-            "converter": "rarity",
-            "path": "data.rarity"
-        },
-        "source": "data.source"
-    }
-}

+{
+    "entries": {
+        "Abacus": {
+            "name": "Abakus",
+            "description": "<p>Der Abakus, auch Z&auml;hlrahmen genannt, ist ein Rechenwerkzeug.</p>",
```

Comment:
Da das alles UTF-8 ist, sollte `ä` hoffentlich auch funktionieren, und `&auml;` sollte nicht notwendig sein.
Kein großes Ding, aber macht Suche etc. leichter. :)

Reply:
werde ich noch ändern, stammt noch von der ersten speicherung in der UTF-8 nicht sauber unterstütz wurde

Reply:
#done